site-data.ftd

```diff
@@ -19,4 +19,7 @@
 -- string terms-url: /terms/
 -- string privacy-url: /privacy/
 
+-- string blog-homepage-url: /scenarios/blogs/
+-- string blog-editor-url: /scenarios/blogs/editor/
```

Comment:
You have to add these to `scenario-site.ftd` as well. 

Also the blog post content should not be in this repo, but in `dotcom` repo. This should only have UI components used for creating blog page, blog index page etc (and maybe a few sample blog post in scenario, so we can see how things look, but these are like lorem-ipsum posts, not real posts).